src/Microsoft.DocAsCode.MarkdownLite/JsonTokenTreeRenderer.cs

```diff
@@ -273,6 +278,15 @@ private string GetSize()
             return "1";
         }
 
+        private string escape(string content)
+        {
+            return content.Replace("\n", "\\n")
+                .Replace("\"", "&quot;")
+                .Replace("<", "&lt;")
+                .Replace(">", "&gt;")
+                .Replace("'", "&#39;");
```

Comment:
can change to use JsonUtility to serialize later, then you needn't to handle the JSON format.

---
In reply to: [75989168](https://github.com/dotnet/docfx/pull/631#discussion_r75989168) [](ancestors = 75989168,75988652)